backend/development.ini.sample

```diff
@@ -1,267 +1,373 @@
-email.notification.content_update.template.html = %(here)s/tracim_backend/templates/mail/content_update_body_html.mak
-email.notification.content_update.template.text = %(here)s/tracim_backend/templates/mail/content_update_body_text.mak
-email.notification.created_account.template.html = %(here)s/tracim_backend/templates/mail/created_account_body_html.mak
-email.notification.created_account.template.text = %(here)s/tracim_backend/templates/mail/created_account_body_text.mak
-email.notification.reset_password_request.template.html = %(here)s/tracim_backend/templates/mail/reset_password_body_html.mak
-email.notification.reset_password_request.template.text = %(here)s/tracim_backend/templates/mail/reset_password_body_text.mak
-# Note: items between { and } are variable names. Do not remove / rename them
 email.notification.content_update.subject = [{website_title}] [{workspace_label}] {content_label} ({content_status_label})
 email.notification.created_account.subject = [{website_title}] Created account
-# processing_mode may be sync or async
-email.notification.processing_mode = sync
-email.notification.smtp.server = your_smtp_server
-email.notification.smtp.port = 25
-email.notification.smtp.user = your_smtp_user
-email.notification.smtp.password = your_smtp_password
-
-## Email sending configuration
-# processing_mode may be sync or async,
-# with async, please configure redis below
-email.processing_mode = sync
-# email.async.redis.host = localhost
-# email.async.redis.port = 6379
-# email.async.redis.db = 0
 
-# Email reply configuration
+### Templates ###
+email.notification.content_update.template.html = %(email.template_dir)s/content_update_body_html.mak
+email.notification.content_update.template.text = %(email.template_dir)s/content_update_body_text.mak
+email.notification.created_account.template.html = %(email.template_dir)s/created_account_body_html.mak
+email.notification.created_account.template.text = %(email.template_dir)s/created_account_body_text.mak
+email.notification.reset_password_request.template.html = %(email.template_dir)s/reset_password_body_html.mak
+email.notification.reset_password_request.template.text = %(email.template_dir)s/reset_password_body_text.mak
+
+####
+# EMAIL-REPLY
+####
+
+# Email reply configuration:
+# Reply by email feature: allow user to answer directly to tracim in reply
+# to notification.
 # This need api.key with non-empty value
+# This feature need additional daemons: see README for more info
+## available choice: True or False
 email.reply.activated = False
+
+### IMAP ###
 email.reply.imap.server = your_imap_server
 email.reply.imap.port = 993
 email.reply.imap.user = your_imap_user
 email.reply.imap.password = your_imap_password
 email.reply.imap.folder = INBOX
-email.reply.imap.use_ssl = true
-email.reply.imap.use_idle = true
-# Re-new connection each 10 minutes
+email.reply.imap.use_ssl = True
+email.reply.imap.use_idle = True
```

Comment:
This parameter is obscure, add a comment to say what is it.

Reply:
@buxx it is good for you:

# IMAP IDLE feature is for real-time notifications (its not necessary to use function send & receive)
# if this option is available in your imap server, change this parameter to "True"

Reply:
yes